telegram/__init__.py

```diff
@@ -242,6 +252,19 @@
     "warnings",
 )
 
+from telegram._stars import (
```

Comment:
the rest of the import statements uses `from ._*` and is sorted alphabetically. If possible, let's try to stay in that schema :)